setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.5.13
+current_version = 0.6.0
```

Comment:
No breaking changes, but big new functionality. 
from semver.org:
 > MINOR version when you add functionality in a backwards-compatible manner, and